internal/service/streamconnection/model_stream_connection_test.go

```diff
@@ -212,6 +218,11 @@ func TestStreamConnectionsSDKToTFModel(t *testing.T) {
 							Protocol:                admin.PtrString(securityProtocol),
 							BrokerPublicCertificate: admin.PtrString(DummyCACert),
 						},
+						Networking: &admin.StreamsKafkaNetworking{
+							Access: &admin.StreamsKafkaNetworkingAccess{
+								Type: admin.PtrString(networkingType),
```

Comment:
could we have a unit test that also checks correct mapping of the `access.name` property? mainly since it is not defined in acceptance tests either.

Reply:
Included `name` into the unit tests. Thanks